### PR TITLE
wait for rails to really be running before we try to do stuff

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

**Comment by jmontleon** [comment]
should fix 503 errors when seeding

File: hooks/post/10-setup_provisioning.rb [hunk 1]
```diff
@@ -52,7 +52,19 @@ def run_rails
     system("echo ':restrict_registered_smart_proxies: false' >> #{DEPLOYMENT_DIR}/foreman/config/settings.yaml")
 
     run_rails
-    say "Rails is running in PID: #{rails_pid}"
+    say "Waiting up to 60 seconds for 200 response code"
```
Comment (jwmatthews) [inline hooks/post/10-setup_provisioning.rb]: How about increasing timeout to 5 minutes?
I've seen times when I wait ~2 minutes for /users/login to render.

**Comment by jmontleon** [comment]
@jwmatthews changed to 5 minutes.

**Comment by jwmatthews** [comment]
ACK

**Pull request merged**